packages/sdk/shell/src/panels/SpacePanel/steps/SpaceManager.tsx

```diff
@@ -80,6 +80,22 @@ export const SpaceManager = (props: SpaceManagerProps) => {
         }
       }, [space]),
     },
+    invitePersistent: {
```

Comment:
ok re question above i'll ask the product team what they have in mind about the need for the user to make this decision.

Reply:
i think there's also some decisionmaking/intent on the app developer as well. and as you noted, some interaction between the invitation parameters, e.g. is an invitation requiring authentication useful to be persistent?

i think it may be simpler to make all invitations persistent, but then i worry about them accumulating, and how to make it clear to the user that there's outstanding invitations, as much as those are separate concerns.